### PR TITLE
Removed double occurrence of 'it' from sentences.

### PR DESCRIPTION
**Pull request by sanganinamrata**
### Description

* Removed double occurrence of **'it'** from sentences.
* Correct grammar mistakes in sentences. Replaced **Is is** to **It is**.
* Fixed **wrong input string into csv** of **Magento_Paypal** module for **en_US** locale.

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios

N/A

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @NamrataChangani. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on Pull Request changes
- `@magento-engcom-team give me new test instance` - deploy **NEW** test instance based on Pull Request changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy Vanilla Magento instance for Issue or Pull Request

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Comment by ishakhsuvarov** [comment]
`/combine 16241 16244`

**Comment by magento-engcom-team** [comment]
Hi @ishakhsuvarov. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@NamrataChangani all pull requests have been successfully combined together: 
 - merged https://github.com/magento/magento2/pull/16241: Correct grammar mistakes in sentences.
 - merged https://github.com/magento/magento2/pull/16244: Fixed wrong input string into csv.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=16240) <br/>All committers have signed the CLA.

File: lib/web/tiny_mce/plugins/autosave/editor_plugin_src.js [hunk 1]
```diff
@@ -20,14 +20,14 @@
...
- * to be working in Firefox 3 and Safari 3.2, but in reality is flaky in those browsers.  As
+ * to be working in Firefox 3 and Safari 3.2, but in reality it is flaky in those browsers.  As
  * HTML 5 gets wider support, the AutoSave plugin will use it automatically. In Windows Vista/7,
  * localStorage is stored in the following folder:
  * C:\Users\[username]\AppData\Local\Microsoft\Internet Explorer\DOMStore\[tempFolder]
  * 
  * 2. sessionStorage - A new feature of HTML 5, sessionStorage works similarly to localStorage,
  * except it is designed to expire after a certain amount of time.  Because the specification
- * around expiration date/time is very loosely-described, it is preferrable to use locaStorage and
+ * around expiration date/time is very loosely-described, it is preferable to use locaStorage and
```
Comment (orlangur) [inline lib/web/tiny_mce/plugins/autosave/editor_plugin_src.js:30]: Interesting, the whole article on this matter :) https://www.grammarly.com/blog/preferably-preferrably/

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-2057](https://jira.corp.magento.com/browse/ENGCOM-2057) has been created to process this Pull Request

File: lib/web/tiny_mce/plugins/autosave/editor_plugin_src.js (hunk 1 above)
Comment (sanganinamrata) [reply lib/web/tiny_mce/plugins/autosave/editor_plugin_src.js:30]: Yes, quite interesting :)

**Comment by magento-engcom-team** [comment]
Hi @NamrataChangani. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.2.6 release.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**